js/ui/components/clubhouse.js

```diff
@@ -505,6 +505,7 @@ class ClubhouseNotificationBanner extends MessageTray.NotificationBanner {
                                onUpdate: () => _clipToMonitor(this.actor),
                                onComplete: () => {
                                    // Ensure it only slides in once
+                                   /this.actor.remove_clip();
```

Comment:
This won't run due to the extra `/`

Reply:
comment/uncomment problem. Now it should be fixed.